snippets/Google.Storage.V1.Snippets/StorageClientSnippets.cs

```diff
@@ -39,7 +39,7 @@ public void Overview()
         {
             var projectId = _fixture.ProjectId;
 
-            // <*>
+            // <Overview>
```

Comment:
what happens if I have an Overview method on my client lib? by picking a name that can't be in the library, we avoid that issue.